src/utilities.rs

```diff
@@ -8,13 +8,63 @@ impl Absorb<CounterAddOp> for i32 {
         *self += operation.0;
     }
 
-    fn absorb_second(&mut self, operation: CounterAddOp, _: &Self) {
-        *self += operation.0;
+    fn sync_with(&mut self, first: &Self) {
```

Comment:
I think this was because it was the same as `absorb_first`, meaning it was identical to the default implementation, meaning this would have the tests also cover those. Not really important, can be re-added.